hack/setup_prow_env.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+# This file should be sourced in a wrapper script (i.e apply_folder.sh)
+
+echo
+echo "[INFO] Running in OpenShift CI (Prow)."
+echo "[INFO] Setting up for PROW"
+echo
+
+# Validate some mandatory variables
+if [ -z "${ART_NAME:-}" ]; then
+    error_exit "missing mandatory variable \$ART_NAME"
+fi
+
+if [ -z "${TF_FOLDER:-}" ]; then
+    error_exit "missing mandatory variable \$TF_FOLDER"
+fi
+
+if [ -z "${TF_VARS:-}" ]; then
```

Comment:
Why? you can apply with using default values (in variables.tf)

Reply:
Correct. Ill update